### PR TITLE
feature: add new playground events

### PR DESCRIPTION
**Pull request by anovazzi1**
this pull request extend the messages funcionalities and create new types of events in the playground

### CODE REVIEW

**Review by Cristhianzl — APPROVED** [review]
lgtm

File: src/frontend/src/components/cardComponent/index.tsx [hunk 1]
```diff
@@ -177,12 +177,12 @@ export default function CollectionCardComponent({
...
                   Playground
                 </Button>
-              )}
+              )} */}
             </div>
           </div>
         </CardFooter>
       </Card>
-      {openPlayground && (
+      {/* {openPlayground && (
```
Comment (Cristhianzl) [inline src/frontend/src/components/cardComponent/index.tsx:185]: can we remove these comments?

File: src/frontend/src/components/storeCardComponent/index.tsx [hunk 2]
```diff
@@ -316,7 +316,7 @@ export default function StoreCardComponent({
           </div>
         </CardFooter>
       </Card>
-      {openPlayground && (
+      {/* {openPlayground && (
```
Comment (Cristhianzl) [inline src/frontend/src/components/storeCardComponent/index.tsx:319]: can we remove these comments?

File: src/frontend/src/components/storeCardComponent/index.tsx [hunk 3]
```diff
@@ -207,7 +207,7 @@ export default function StoreCardComponent({
         <CardFooter>
           <div className="z-50 flex w-full items-center justify-between gap-2">
             <div className="flex w-full flex-wrap items-end justify-end gap-2">
-              {playground && (
+              {/* {playground && (
```
Comment (Cristhianzl) [inline src/frontend/src/components/storeCardComponent/index.tsx:210]: can we remove these comments?

File: src/frontend/src/modals/IOModal/components/chatView/chatInput/components/buttonSendWrapper/newButtonSendWrapper.tsx [hunk 4]
```diff
@@ -60,24 +61,27 @@ const ButtonSendWrapper = ({
...
-        <div className="flex items-center gap-2">
+        <div className="flex items-center gap-2 rounded-md text-[14px] font-medium">
           Stop
-          <Loading className="text-black" />
+          <Loading className="h-[16px] w-[16px]" />
         </div>
       </Case>
 
-      <Case condition={showPlayButton}>
+      {/* <Case condition={showPlayButton}>
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/chatView/chatInput/components/buttonSendWrapper/newButtonSendWrapper.tsx:73]: can we remove these comments?

File: src/frontend/src/modals/IOModal/components/chatView/chatMessage/components/editMessageField/newEditMessageField.tsx [hunk 5]
```diff
@@ -25,52 +26,52 @@ export default function EditMessageField({
...
         ref={textareaRef}
-        className="h-mx-full w-full resize-none border-0 bg-zinc-800 focus:ring-0"
-        onBlur={() => {
-          if (!isButtonClicked) {
-            onCancel();
-          }
-        }}
+        className="max-h-[400px] w-full resize-none rounded-none border-0 bg-muted shadow-none focus:ring-0"
+        // onBlur={() => {
+        //   if (!isButtonClicked) {
```
Comment (Cristhianzl) [inline src/frontend/src/modals/IOModal/components/chatView/chatMessage/components/editMessageField/newEditMessageField.tsx:34]: can we remove these comments?

File: src/frontend/src/routes.tsx [hunk 6]
```diff
@@ -201,9 +201,9 @@ const router = createBrowserRouter(
                 </Route>
                 <Route path="view" element={<ViewPage />} />
               </Route>
-              <Route path="playground/:id/">
+              {/* <Route path="playground/:id/">
```
Comment (Cristhianzl) [inline src/frontend/src/routes.tsx:204]: can we remove these comments?

**Review by Cristhianzl — COMMENTED** [review]
lgtm

File: src/frontend/src/components/cardComponent/index.tsx (hunk 1 above)
Comment (anovazzi1) [reply src/frontend/src/components/cardComponent/index.tsx:185]: we probably will bring this feature back in a near future, that's why I want to keep it commented

**Pull request merged**